src/internal/linux/AudioDestinationLinux.h

```diff
@@ -0,0 +1,55 @@
+// License: BSD 3 Clause
+// Copyright (C) 2010, Google Inc. All rights reserved.
+// Copyright (C) 2015+, The LabSound Authors. All rights reserved.
+
+#ifndef AudioDestinationLinux_h
+#define AudioDestinationLinux_h
+
+#include "LabSound/core/AudioNode.h"
+#include "LabSound/core/AudioBus.h"
+
+#include "internal/AudioDestination.h"
+
+#include "rtaudio/RtAudio.h"
+#include <iostream>
+#include <cstdlib>
+
+namespace lab {
+
+class AudioDestinationLinux : public AudioDestination 
+{ 
+
+public:
+
+    AudioDestinationLinux(AudioIOCallback &, float sampleRate);
+    virtual ~AudioDestinationLinux();
+
+    virtual void start() override;
+    virtual void stop() override;
+
+    bool isPlaying() override { return m_isPlaying; }
```

Comment:
Do we ever use the `isPlaying()` api?

Reply:
It doesn't seem to be part of the webaudio spec.